samples/CGCatalog/CGCatalog/Samples/CGCCGContextBeginEndTransparencyLayer.m

```diff
@@ -48,7 +47,7 @@ - (void)drawRect:(CGRect)rect {
     // Set composite alpha of the transparency layer
     // CGContextSetAlpha(context, 0.5);
```

Comment:
I just tested again after the final implementation of the transparency APIs, and this issue no longer exists; it seems to use the Alpha like it should.  I'll remove this one, as it's not much different than using SetHue as a test.